lighthouse/run.sh

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 
-while ! curl "http://${NODE}:3600/eth/v1/node/health" 2>/dev/null; do
-  echo "Waiting for http://${NODE}:3600 to become available..."
+apt-get update && apt-get install -y curl jq wget
```

Comment:
so this is introduces a delay of 1m to every time you start the  lighthouse container. Not sure this optimisation is worth it.